### ISSUE TITLE
Multiple Issues

### ISSUE THREAD

**Issue by SkididSkider** [rei-kes/Fedoraware#17]
Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.

This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)

I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.

Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.

**Comment by karawaya** [comment]
Yes, the crashes are relatable. Nothing u can't really do about it till  rei does something about it. However I disagree with u on the dt part. Dt works fine for me. U prob have a crappy config. Dunno bout the back track and prop though never had these issues.

**Comment by phonkv2** [comment]
> Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.
> 
> 
> 
> This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)
> 
> 
> 
> I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.
> 
> 
> 
> Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.

you can try using interp with backtrack works fine for me tho dt also works fine for me are you sure you know how to use dt it might be a config issue

**Comment by SkididSkider** [comment]
> > Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.
> > This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)
> > I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.
> > Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.
> 
> you can try using interp with backtrack works fine for me tho dt also works fine for me are you sure you know how to use dt it might be a config issue

Mind if you send me your cfg to see what i messed up?

**Comment by pay2lose2** [comment]
> "The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons."
You probably have it set as "Last" which means it will only work on the last model, there should be an option for "All" which will work between the enemy and the last

> "Now the DT works but it's either I am shit at configing it or sometimes it ain't working."
Most likely config issue, the default settings for DT should be fine

**Comment by phonkv2** [comment]
> > > Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.
> 
> > > This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)
> 
> > > I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.
> 
> > > Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.
> 
> > 
> 
> > you can try using interp with backtrack works fine for me tho dt also works fine for me are you sure you know how to use dt it might be a config issue
> 
> 
> 
> Mind if you send me your cfg to see what i messed up?

yeah sure here 

https://mega.nz/file/wMVlwR5R#cvilkAn9__afhafAiHxX0rK0YM6d9LYL-uWt5HL1vQk

**Comment by godofateris** [comment]
> > > > Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.
> > 
> > 
> > > > This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)
> > 
> > 
> > > > I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.
> > 
> > 
> > > > Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.
> > 
> > 
> > > 
> > 
> > 
> > > you can try using interp with backtrack works fine for me tho dt also works fine for me are you sure you know how to use dt it might be a config issue
> > 
> > 
> > Mind if you send me your cfg to see what i messed up?
> 
> yeah sure here
> 
> https://mega.nz/file/wMVlwR5R#cvilkAn9__afhafAiHxX0rK0YM6d9LYL-uWt5HL1vQk

my man its a not best,but great cfg for this cheat. Thank u for sharing

**Comment by phonkv2** [comment]
> > > > > Soo let's start with the crashing, as in another issue thread someone said that I randomly crash and yes this is true. It happens often for me and it's kinda annoying. The backtrack is ass, I can't stab with Spy only if I go back to where the first model of the character appears and it's the same with hitscan weapons. Now the DT works but it's either I am shit at configing it or sometimes it ain't working.
> 
> > > 
> 
> > > 
> 
> > > > > This ain't an issue or it is(idk) but the prop color doesn't apply when i change it.(not really an issue for me because i switched from xarQon gay femboy colors)
> 
> > > 
> 
> > > 
> 
> > > > > I do have hope for this cheat and it's really good. I wonder how it's gonna be in the future.
> 
> > > 
> 
> > > 
> 
> > > > > Also i would like if you would add a line to the crit hack(from the ateris leak) it looks so good for some reason.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > you can try using interp with backtrack works fine for me tho dt also works fine for me are you sure you know how to use dt it might be a config issue
> 
> > > 
> 
> > > 
> 
> > > Mind if you send me your cfg to see what i messed up?
> 
> > 
> 
> > yeah sure here
> 
> > 
> 
> > https://mega.nz/file/wMVlwR5R#cvilkAn9__afhafAiHxX0rK0YM6d9LYL-uWt5HL1vQk
> 
> 
> 
> my man its a not best,but great cfg for this cheat. Thank u for sharing

yeah it could use a few improvements like visuals but i keep forgetting to change it lol
